siad/main.go

```diff
@@ -132,6 +132,9 @@ Explorer (e):
 
 // main establishes a set of commands and flags using the cobra package.
 func main() {
+	if build.DEBUG {
+		fmt.Println("Running with debugging enabled")
+	}
```

Comment:
alternatively, we could add `-debug` to the version string printed by `siad version`.